playbooks/aws/openshift-node/terminate.yml

```diff
@@ -0,0 +1 @@
+../openshift-master/terminate.yml
```

Comment:
This looks odd to have node playbook just call master playbook.  If not an error, you may want to add comment.